GameStar GamePro.js

```diff
@@ -0,0 +1,160 @@
+{
+	"translatorID": "c87a910d-70aa-4f33-9bec-f703d63ba84f",
+	"label": "GameStar/GamePro",
+	"creator": "Matthias Mailänder",
+	"target": "^https?://(www\\.)?game(star|pro)\\.de",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-04-08 10:44:54"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Matthias Mailänder
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (url.includes('/artikel/')) {
+		return "magazineArticle";
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "magazineArticle") {
+		scrape(doc, url);
+	}
+}
```

Comment:
Could you add a `multiple` case? The "web translator template" generated by Scaffold will give you 95% of the code you need - you'll probably just have to tweak the selectors.

Reply:
If `multiple` means grabbing search results, then I don't see the purpose of those.

Reply:
A lot of the time, neither do I. But almost every Zotero web translator supports them, unless its target site doesn't have useful search results. That's not the case here, so they should be supported. It'll seriously be ten simple lines of code from the template.

Reply:
So essentially, you want me to write a parser for stuff like https://www.gamestar.de/suche/?query=grand+theft+auto just like
https://github.com/zotero/translators/blob/2b2de18fad702214f00c292d394c83f644f4e7e5/PC%20Gamer.js#L76-L90
Sadly the HTML/CSS on that site is a lot messier.

Reply:
I used 
https://github.com/zotero/translators/blob/1d1ce6111ac17a4064c1d8836f6beec54404c32b/PC%20Gamer.js#L76-L90
as a reference and copied the selectors using the Chrome developer tools.